src/test/java/AddOwnerPageTest.java

```diff
@@ -116,4 +116,33 @@ public void canAddOwnerAndAnimal() {
         webApp.addPetPage().addNewPet("Sarah", "10", Type.DOG);
         webApp.ownerInformationPage().verifySecondHeaderText("Pets and Visits");
     }
+    @Test
+    @DisplayName("PC-24: Can add owner with special characters and numbers mixed in all fields without telephone")
+    public void canAddOwnerWithSpecialCharactersAndNumbersMixedInAllFieldsWithoutTelephone() {
+        webApp.components().mainMenu().clickFindOwnersLink();
+        webApp.findOwnersPage().clickAddOwnerLink();
+        webApp.addOwnerPage().enterFirstName("@nton098");
+        webApp.addOwnerPage().enterLastName("$hehov123");
+        webApp.addOwnerPage().enterAddress("123#^&ABV\n");
+        webApp.addOwnerPage().enterCity("*![]<yooohoho>{}123/\\\n");
+        webApp.addOwnerPage().enterTelephone("0000");
+        webApp.addOwnerPage().clickAddOwnerButton();
+        webApp.ownerInformationPage().verifyHeaderText("Owner Information");
```

Comment:
This is more of a bug. We dont expect this kind of behavior to be ok.
We'd rather want to get error message here.